src/data/zsqrtd/basic.lean

```diff
@@ -85,6 +85,19 @@ def conj : ℤ√d → ℤ√d
 @[simp] theorem conj_im : ∀ z : ℤ√d, (conj z).im = -z.im
 | ⟨x, y⟩ := rfl
 
+@[simp] lemma conj_zero : conj (0 : ℤ√d) = 0 :=
+by rw [ext, conj_re, conj_im, zero_im, zero_re, neg_zero, and_self]
```

Comment:
Can you use this to show that `conj` is an `add_monoid_hom`?

That is, declare
```
/-- `conj` as an `add_monoid_hom`. -/
def conj_hom : ℤ√d →+ ℤ√d :=
{ to_fun := conj,
  map_add' := λ ⟨a, ai⟩ ⟨b, bi⟩, ext.mpr ⟨rfl, neg_add _ _⟩,
  map_zero' := ext.mpr ⟨rfl, neg_zero⟩ }
```
And then prove `conj_zero`, `conj_neg`, `conj_add`, `conj_sub` as `conj_hom.map_zero` etc.

Reply:
Done. (I had to move the `conj` section below the `add_monoid` instance.)